api/groups_test.go

```diff
@@ -767,16 +789,46 @@ func TestCreateNewGroup(t *testing.T) {
 					So(errorResponse, ShouldBeNil)
 				},
 			},
-			// 400 response - group already exists
```

Comment:
I don't understand why this test was removed 
surely that a test for whether a group already exists is valid 
and that response is more meaningful to a user...